src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -139,7 +148,14 @@ internal CodeGenIL(IDictionary<string, IList> input, string filePath, ProtoCore.
                 }
                 EmitOpCode(OpCodes.Ret);
                 return (asm, type);
+#if DEBUG
             }
+#endif
+            EmitOpCode(OpCodes.Ret);
+#if DEBUG
+            writer.Close();
```

Comment:
why do we need to call writer.Close() if the writer is wrapped in a using?

Reply:
I think this line reappeared after I resolved merge conflicts with my local branch, will remove it.